injector/src/main/java/edu/ucr/cs/riple/injector/changes/AddMarkerAnnotation.java

```diff
@@ -62,7 +62,17 @@ Modification computeTextModificationOn(T node) {
     if (annotAlreadyExists) {
       return null;
     }
-    return new Insertion(annotationExpr.toString(), range.begin);
+
+    String annotation = annotationExpr.toString();
```

Comment:
I don't think this is a right place to add your changes. Your changes should alter [AddTypeUseMarkerAnnotation](https://github.com/ucr-riple/NullAwayAnnotator/blob/master/injector/src/main/java/edu/ucr/cs/riple/injector/changes/AddTypeUseMarkerAnnotation.java)  behavior. 

**Hint**: all you changes should be in [here](https://github.com/ucr-riple/NullAwayAnnotator/blob/c965c90891c181c29a21c4bf8731946422be4f73/injector/src/main/java/edu/ucr/cs/riple/injector/Helper.java#L611) where we are returning array component range.

Reply:
Done